apps/web/pages/index.tsx

```diff
@@ -1,13 +1,63 @@
-import Link from "next/link";
-import { Button } from "ui";
+import Head from "next/head";
+import Header from "../components/Header";
+import HeroSection from "../components/HeroSection";
+import Stats from "../components/Stats";
+import TemplatesSection from "../components/TemplatesSection";
+import Contributors from "../components/Contributors";
+import Footer from "../components/Footer";
+import getAllTemplates, { TemplatesInterface } from "../utils/getAllTemplates";
+import {
+  getStats,
+  getContributors,
+  StatsInterface,
+  ContributorsInterface,
+} from "../utils/getStatsAndContributors";
 
-export default function Web() {
+interface WebProps {
+  stats: StatsInterface;
+  contributors: ContributorsInterface;
+  templates: TemplatesInterface;
+}
+
+export default function Web({ stats, contributors, templates }: WebProps) {
```

Comment:
To simplify naming, private interfaces (which you don't export out of the component for some reason. Can be simply named Props